ts/models/IdplLoginPhase.ts

```diff
@@ -0,0 +1,10 @@
+export type IdplLoginPhase = {
+  id: string;
+  detail: string;
+};
+
+export type IdpLoginEnd = {
+  id: string;
+  duration: number;
```

Comment:
always specify the unit for `number` props/variables - we also have [types for time units](https://github.com/teamdigitale/io-ts-commons/blob/master/src/units.ts#L9) do avoid mistakes (i.e. passing seconds to a function that wants milliseconds) 

```suggestion
  durationMillis: number;
```